Week3/homework/CreditCardValidate.js

```diff
@@ -0,0 +1,46 @@
+"use strict"
+
+function validCreditCard (num) {
+    let card = num;
+    let cardNumber = card.split("");
+    let cardNumberDif = cardNumber.filter((este, i) => cardNumber.indexOf(este) === i);
+    let cardNumberLen = cardNumberDif.length;
+    let soma = 0;
```

Comment:
I am not sure what are you trying to achieve here. This is firstly because the variable names you are using are not quite descriptive(`cardNumberDif`, `cardNumberLen` and `soma`) about what values those variables would potentially be holding to.

Also, not sure if I understood correctly your logic of the `filter` operation. What is it that you are trying to achieve?